agent/tests/test_input/test_2/test_mysql_http.py

```diff
@@ -39,6 +43,9 @@ def test_create_advanced(self, cli_runner, name, source):
                                    input=f'{source}\n{name}\nSELECT * FROM test WHERE {{TIMESTAMP_CONDITION}} AND not country is NULL\n\n86400\n{days_to_backfill}\n1\ntimestamp_unix\nunix\ny\ntest\nclicks:gauge impressions:gauge\nadsize country\nkey1:val1 key2:val2\n\n\n\n\n')
         assert result.exit_code == 0
 
+    def test_create_events_pipeline_with_file(self, cli_runner, file_name, override_config: dict):
+        super(TestMySQL, self)._test_create_with_file(cli_runner, file_name, override_config, cli.pipeline.create_events)
```

Comment:
Why do you prefer `super(TestMySQL, self)._test_create_with_file`?
Because there is no overriding, you can call just `self._test_create_with_file()`